core-web/libs/edit-content/src/lib/fields/dot-edit-content-radio-field/dot-edit-content-radio-field.component.spec.ts

```diff
@@ -0,0 +1,233 @@
+import { Spectator, createComponentFactory } from '@ngneat/spectator';
+
+import { ControlContainer, FormGroupDirective } from '@angular/forms';
+
+import { RadioButton } from 'primeng/radiobutton';
+
+import { DotEditContentRadioFieldComponent } from './dot-edit-content-radio-field.component';
+
+import {
+    RADIO_FIELD_BOOLEAN_MOCK,
+    RADIO_FIELD_FLOAT_MOCK,
+    RADIO_FIELD_INTEGER_MOCK,
+    RADIO_FIELD_TEXT_MOCK,
+    createFormGroupDirectiveMock
+} from '../../utils/mocks';
+
+describe('DotEditContentRadioFieldComponent', () => {
+    let spectator: Spectator<DotEditContentRadioFieldComponent>;
+
+    const createComponent = createComponentFactory({
+        component: DotEditContentRadioFieldComponent,
+        componentViewProviders: [
+            { provide: ControlContainer, useValue: createFormGroupDirectiveMock() }
+        ],
+        providers: [FormGroupDirective],
+        detectChanges: false
+    });
+
+    beforeEach(() => {
+        spectator = createComponent();
+    });
+    it('should if have an object without value and label, the label must be the same as the value, and of the same type', () => {
+        const RADIO_FIELD_TEXT_MOCK_WITHOUT_VALUE_AND_LABEL = {
+            ...RADIO_FIELD_TEXT_MOCK,
+            values: 'Dot'
+        };
+        spectator.setInput('field', RADIO_FIELD_TEXT_MOCK_WITHOUT_VALUE_AND_LABEL);
+        spectator.detectComponentChanges();
+
+        const expectedList = [
+            {
+                label: 'Dot',
+                value: 'Dot'
+            }
+        ];
+
+        expectedList.forEach((option) => {
+            expect(
+                spectator
+                    .queryAll(RadioButton)
+                    .find((radioOption) => radioOption.value === option.value)
+            ).toBeTruthy();
+        });
```

Comment:
```suggestion
            expect(spectator.queryAll(RadioButton).map((radio) => radio.value)).toEqual([
                'one',
                'two'
            ]);
```

Reply:
Also, we need to test the `inputId` and the label with the `for` (to test the semantic relationship) and the label `innerText`.

The same applies for all other cases but labels, labels can be tested just once.